app/models/concerns/catkey.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Catkey
+  def self.deserialize(model)
+    Array(model.identification.catalogLinks).select { |link| link.catalog == 'symphony' }.map(&:catalogRecordId).join(', ')
```

Comment:
Not a giant deal, but I wonder if it's worth cou can replace the double iteration here with `#filter_map`?

Reply:
I'm old skool!

Reply:
Change made.